BPCompatibleAlertControllerDemo/CompatibleAlertControllerDemo/ViewController.swift

```diff
@@ -9,52 +9,69 @@
 import UIKit
 
 class ViewController: UIViewController {
-    var alertController: BPCompatibleAlertController?
 
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        var observerObject: NSObjectProtocol?
+        
+        let cleanupAfterAlertDismissed: (Void) -> Void =
```

Comment:
Could you please explain the use of this? Does this mean that we're requiring the user to have to manually remove the observer in each action specified (if they do add an observer)?